Middlewares/GlobalExceptionHandlerMiddleware.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Azure.Functions.Worker.Middleware;
+using Microsoft.Extensions.Logging;
+using System.Net;
+
+namespace Middlewares
+{
+    public class GlobalExceptionHandlerMiddleware : IFunctionsWorkerMiddleware
+    {
+        private readonly ILogger<GlobalExceptionHandlerMiddleware> _logger;
+
+        public GlobalExceptionHandlerMiddleware(ILogger<GlobalExceptionHandlerMiddleware> logger)
+        {
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        }
+        public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next)
+        {
+			try
```

Comment:
can you fix the indentation please?

Reply:
Ctrl K + Ctrl D!! haha

Reply:
Can you Verify that it's an identation problem?
I don't see it locally 😞